ts-packages/web/src/app/teams/[username]/_components/team-profile.tsx

```
@@ -53,15 +53,15 @@ export default function TeamProfile({ team }: TeamProfileProps) {
+          <div className="w-20 h-20 rounded-full bg-profile-bg" />
         )}
       </div>
 
-      <div className="font-medium">{team.nickname}</div>
+      <div className="font-medium text-text-primary">{team.nickname}</div>
 
       <div
         id="user-profile-description"
-        className="text-xs text-gray-400"
+        className="text-xs text-desc-text"
         dangerouslySetInnerHTML={{ __html: team.html_contents }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)